schema/deploy/tables/project.sql

```diff
@@ -54,5 +55,6 @@ comment on column cif.project.project_status_id is 'The id of the project_status
 comment on column cif.project.total_funding_request is 'The total amount of funding requested for the project';
 comment on column cif.project.sector_name is 'The industry sector this project relates to';
 comment on column cif.project.additional_sector_information is 'Any additional information about this project in reference to the industry sector such as a sub-sector';
+comment on column cif.project.comments is 'Historical amendment data for existing projects to track why and when decisions were made';
```

Comment:
This should just say something like "General comments pertaining to the project" The fact that it's going to be used for historical amendments data is not relevant to the data model

Reply:
Done